examples/models/llama3_2_vision/text_decoder/test/test_text_decoder.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Export and ExecuTorch tests for CLIP vision encoder are covered by test_models.sh.
+# Only test AOTI in this file
+import json
+import os
+import tempfile
+import unittest
+
+import torch
+
+from executorch.examples.models.llama3_2_vision.text_decoder.model import (
+    Llama3_2Decoder,
+)
+from torch.testing import assert_close
+
+params = {
```

Comment:
Can load https://github.com/pytorch/executorch/blob/main/examples/models/llama3_2_vision/text_decoder/params/demo_config.json, also can set n_layers to 1 to make test go faster

Reply:
Changed, reading json is not very convenient in unit test